### PR TITLE
LPS-192564 | Master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
https://liferay.atlassian.net/browse/LPS-192564

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/dxp/apps/document-library-opener/document-library-opener-onedrive-test/src/testFunctional/tests/OneDrive.testcase [hunk 1]
```diff
@@ -812,7 +812,7 @@ definition {
 			dmDocumentVersionNumber = "2.0");
 	}
 
-	@description = "This test ensures that a Word document title can be changed via the OneDrive editor."
+	@description = "This test covers LPS-191439. This test ensures that a Word document title can be changed via the OneDrive editor."
```
Comment (Jeremy-Ch) [inline modules/dxp/apps/document-library-opener/document-library-opener-onedrive-test/src/testFunctional/tests/OneDrive.testcase:815]: Please use the correct JIRA ticket.

File: portal-web/test/functional/com/liferay/portalweb/macros/OneDrive.macro [hunk 2]
```diff
@@ -214,6 +214,17 @@ definition {
 		MenuItem.clickNoError(menuItem = "Edit in Office 365");
 	}
 
+	macro gotoEditInPreviewPage {
```
Comment (Jeremy-Ch) [inline portal-web/test/functional/com/liferay/portalweb/macros/OneDrive.macro:217]: Is possible to combine this macro with `OneDrive.gotoEdit`?Just add a condition to the macro.

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
LGTM

**Comment by Jeremy-Ch** [comment]
ci:forward

**Pull request closed without merging**